doc/telestrat.sgml

```diff
@@ -0,0 +1,177 @@
+described in detail in the separate <url url="funcref.html" name="function
+reference">. Even functions marked as "platform dependent" may be available on
+more than one platform. Please see the function reference for more
+information.
+
+
+
+<sect>Binary format<p>
+
+The standard binary output format generated by the linker for the Telemon 3.0 target
+is a machine language program with a 20 bytes header described here : http://orix.oric.org/doku.php?id=orix:header
+
+
+
+<sect>Memory layout<p>
+
+In the standard setup, cc65-generated programs use the memory from
+&dollar;0801 to &dollar;9800; so, nearly 37K of memory (including the stack) is
+available. ROM calls are possible without further precautions.
+
+
+
+Special locations:
+
+<descrip>
+  <tag/Stack/
+  The C runtime stack is located at &dollar;97FF (or &dollar;B3FF), and grows
+  downwards.
+
+  <tag/Heap/
+  The C heap is located at the end of the program, and grows towards the C
+  runtime stack.
+
+</descrip><p>
+
+
+
+<sect>Platform-specific header files<p>
+
+Programs containing Telemon 3.0 -specific code may use the <tt/telemon.h/ header file.
+
+
+<sect1>Telemon 3.0-specific functions<p>
+
+The functions listed below are special for the Telemon 3.0. See the <url
+url="funcref.html" name="function reference"> for declaration and usage.
+
+<itemize>
+<item>explode
+<item>ping
+<item>shoot
+<item>zap
+<item>oupsx
+</itemize>
+
+
+<sect1>Hardware access<p>
+
+The following pseudo variables declared in the <tt/atmos.h/ header file do allow
+access to hardware located in the address space. Some variables are
+structures; accessing the struct fields will access the chip registers.
+
+<descrip>
+
+  <tag><tt/VIA/</tag>
+  Access to the VIA (Versatile Interface Adapter) chip is available via the
+  <tt/VIA/ variable. The structure behind this variable is explained in <tt/_6522.h/.
+
+</descrip><p>
+
+
+
+<sect>Loadable drivers<p>
+
+<em>Note:</em> Since the Atmos doesn't have working disk I/O
+(see <ref id="limitations" name="section &quot;Limitations&quot;">), the
+available drivers cannot be loaded at runtime (so the term "loadable drivers"
+is somewhat misleading). Instead, the drivers have to be statically linked. While
+this may seem overhead, it has two advantages:
+
+
+<sect1>Extended memory drivers<p>
+
+No extended memory drivers are currently available for the Atmos.
+
+
+<sect1>Joystick drivers<p>
+
+<descrip>
+
+telemon 3.0 manages joysticks but it had been handled yet.
+
+</descrip><p>
+
+
+<sect1>Mouse drivers<p>
+
+Telemon 3.0 manages also mouse, but it had been no handled yet in this version.
+
+<sect1>RS232 device drivers<p>
+
+<descrip>
+
+not done
+
+</descrip><p>
+
+
+
+<sect>Limitations<label id="limitations"><p>
+
+<sect1>Disk I/O<p>
+
+This version handles fopen, fread, fclose primitives. Because Telemon 3.0 handles these two primitives. By the way, 
+it uses an extension "ch376 card" which handles sdcard and FAT 32 usb key.
```

Comment:
- We're under `Limitations` here. If disk I/O works as expected then there's no need for such a section at all. If disk I/O isn't fully available (as I understand it's not in 2.4) then this is what needs to be stated here. And if the 2.4 has the same limitations the ATMOS has (with the same "hack" to allow stdin / stdout) then the statements from the ATMOS should be used here.

- I understand that the file I/O we're talking about here requires the "ch376" thingy to provide an actual file system to access. If that is true you certainly need to elaborate on this. Again either here or on orix.oric.org and link to it from here. This requirement has nothing to do with the fact that this file I/O is - as far as I understand - new/uncommon. I.e. the Apple II and the ATARI have several DOSes if not all of them are supported by the cc65 file I/O then this needs to be clarified.